aws_lambda_powertools/utilities/validation/validator.py

```diff
@@ -161,6 +186,10 @@ def validate(
         Alternative JMESPath options to be included when filtering expr
     formats: Dict
         Custom formats containing a key (e.g. int64) and a value expressed as regex or callback returning bool
+    handlers: Dict
+        Custom methods to retrieve remote schemes, keyed off of URI scheme
+    provider_options: Dict
+        Arguments that will be passed directly to the underlying validate call
```

Comment:
same as earlier, let's help customers understand what's available. We wouldn't be able to type `TypedDict`, as we can't control their API.